<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumForm</class>
 <widget class="QDialog" name="SpectrumForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Raw data</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="rawDataTbv"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Spectrum Plot</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Sample</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Sample description</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextEdit" name="sampDescrTed"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Acquisition</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Start of acquisition</string>
            </property>
            <property name="buddy">
             <cstring>startDte</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="startDte">
            <property name="timeSpec">
             <enum>Qt::LocalTime</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Delay in days</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Live time:</string>
            </property>
            <property name="buddy">
             <cstring>leLT_2</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leLT_2"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Real time:</string>
            </property>
            <property name="buddy">
             <cstring>leRT_2</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leRT_2"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Number of channels:</string>
            </property>
            <property name="buddy">
             <cstring>leNoCh_2</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leDelayInDays">
            <property name="text">
             <string>13</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="leNoCh_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Quantity</string>
         </property>
         <layout class="QGridLayout" name="_4">
          <property name="margin">
           <number>8</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="3">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="leUncertainty_2"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Uncertainty:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="cbUnit_2">
            <item>
             <property name="text">
              <string>L</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>g</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>kg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sample</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QLineEdit" name="leValue_2"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Analysis Parameters</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Nuclide libraries</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Analysis library:</string>
            </property>
            <property name="buddy">
             <cstring>leNoFwhmOfPeak</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbxSelectedAnalysisLib"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Calibration library:</string>
            </property>
            <property name="buddy">
             <cstring>leNoFwhmOfSideBaseline</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbxSelectedCalibLib"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Peak Fit</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string># Fwhms for each peak:</string>
            </property>
            <property name="buddy">
             <cstring>leNoFwhmOfPeak</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leNoFwhmOfPeak">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>4,0</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string># Fwhms for each side baseline</string>
            </property>
            <property name="buddy">
             <cstring>leNoFwhmOfSideBaseline</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leNoFwhmOfSideBaseline">
            <property name="text">
             <string>5,0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Function type of baselines:</string>
            </property>
            <property name="buddy">
             <cstring>cbxTypeBaseline</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbxTypeBaseline">
            <item>
             <property name="text">
              <string>Exponential</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Polynomial</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Exponential + constant</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Grade of polynomial for side baselines:</string>
            </property>
            <property name="buddy">
             <cstring>leBaselinePolyGrade</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leBaselinePolyGrade">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Interpolation method for baseline under peaks:</string>
            </property>
            <property name="buddy">
             <cstring>cbxTypeBaseline</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbxInterpolationMethod">
            <item>
             <property name="text">
              <string>Peak-height weight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cubic spline</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Fit strategy</string>
            </property>
            <property name="buddy">
             <cstring>cbxTypeBaseline</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cbxFitStrategy">
            <item>
             <property name="text">
              <string>Peak fit after baseline subtraction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>baseline and peak parameters determination in one step</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Peak identification</string>
         </property>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Tolerance:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="leToler">
            <property name="text">
             <string>0,3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>keV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Energy Calibration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Energy X channel parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxEnChFitType">
           <item>
            <property name="text">
             <string>Polyn. gr. 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Linear</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="enxchanTbv">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="doEnChFitPbt">
           <property name="text">
            <string>Do energy x channel fit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtClearEnergyCalibration">
           <property name="text">
            <string>Clear energy calibration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtAcceptNewEnergies">
           <property name="text">
            <string>Accept new energy values</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>FWHM X channel parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxFwhmChFitType">
           <item>
            <property name="text">
             <string>a0 + a1 * sqrt(En)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>a0 + a1*En + a2*En^2</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="fwhmxenTbv"/>
         </item>
         <item>
          <widget class="QPushButton" name="doFwhmChFitPbt">
           <property name="text">
            <string>Do FWHM x channel fit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Peak Search</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="tlBinFiltGr">
              <property name="text">
               <string>Binomial filter grade</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="leBinFiltGr">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="tlPkcut">
              <property name="text">
               <string>Search sensitivity</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lePkcut">
              <property name="text">
               <string>0,3</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="tlSideChannelsSearch">
              <property name="text">
               <string># of channels on each side:</string>
              </property>
              <property name="buddy">
               <cstring>leSideChannelsSearch</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="leSideChannelsSearch">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Data points:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="energypointsTbv">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtNewSearch">
              <property name="text">
               <string>New search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Efficiency Calibration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Efficiency points - number:</string>
             </property>
             <property name="buddy">
              <cstring>neffptsLne</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="neffptsLne">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="doefficfitPbt">
             <property name="text">
              <string>Do fit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="efficPtsTbv"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Efficiency X energy parameters - Polynomial grade:</string>
             </property>
             <property name="buddy">
              <cstring>polygradeLne</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="polygradeLne">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="effxenTbv"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Nuclide Lib.</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QTableView" name="gammasPerNuclideTbv"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Peak Report</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTableView" name="fittedPeaksTbv"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Activities</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="activitiesTbv"/>
       </item>
       <item>
        <widget class="QTableView" name="partialActivitiesTbv"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Fit Log</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPlainTextEdit" name="pteWidthFitLog"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="pteBaseLinesFitLog"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
